src/test/java/org/datadog/jenkins/plugins/datadog/traces/JobNameWrapperTest.java

```diff
@@ -24,19 +24,19 @@ public static Collection<Object[]> data() {
         sampleMap.put("key2", "value2");
 
         final Object[][] data = new Object[][] {
-                {null, null, null, EMPTY_MAP},
-                {null, "master", null, EMPTY_MAP},
-                {"", "master", "", EMPTY_MAP},
-                {"jobName", null, "jobName", EMPTY_MAP},
-                {"jobName", "", "jobName", EMPTY_MAP},
-                {"jobName", "master", "jobName", EMPTY_MAP},
-                {"jobName/master", "master", "jobName", EMPTY_MAP},
-                {"jobName/another", "master", "jobName/another", EMPTY_MAP},
-                {"jobName/another/branch", "another/branch", "jobName", EMPTY_MAP},
-                {"jobName/another%2Fbranch", "another/branch", "jobName", EMPTY_MAP},
-                {"jobName/KEY1=VALUE1,KEY2=VALUE2", "master", "jobName", sampleMap},
-                {"jobName/KEY1=VALUE1,KEY2=VALUE2/master", "master", "jobName", sampleMap},
-                {"jobName/KEY1=VALUE1,KEY2=VALUE2/another-branch", "master", "jobName", sampleMap}
+                {null, null, EMPTY_MAP},
+                {null, "master", EMPTY_MAP},
+                {"", "master", EMPTY_MAP},
+                {"jobName", null, EMPTY_MAP},
+                {"jobName", "", EMPTY_MAP},
+                {"jobName", "master", EMPTY_MAP},
+                {"jobName/master", "master", EMPTY_MAP},
+                {"jobName/another", "master", EMPTY_MAP},
+                {"jobName/another/branch", "another/branch", EMPTY_MAP},
+                {"jobName/another%2Fbranch", "another/branch", EMPTY_MAP},
+                {"jobName/KEY1=VALUE1,KEY2=VALUE2", "master", sampleMap},
+                {"jobName/KEY1=VALUE1,KEY2=VALUE2/master", "master", sampleMap},
+                {"jobName/KEY1=VALUE1,KEY2=VALUE2/another-branch", "master", sampleMap}
```

Comment:
Do we still need the JobNameWrapper? This is only for configurations? Maybe we should clarify that?

Reply:
I wasn't sure, but I though about turning it into a static function that extracts the configurations from the job name and returns them. Do you like that better than keeping it as is?

Reply:
Yeah, let's do something like that because this class was to encapsulate the logic for jobName extraction, but I guess it's not needed anymore, so we could what you're proposing